forge-gui/res/cardsfolder/upcoming/melt_through.txt

```diff
@@ -0,0 +1,7 @@
+Name:Melt Through
+ManaCost:R
+Types:Instant
+A:SP$ DealDamage | ValidTgts$ Creature,Player,Planeswalker | TgtPrompt$ Select any target | NumDmg$ 2 | SubAbility$ DBEffect | SpellDescription$ Melt Through deals 2 damage to any target. If it's a creature, it perpetually gains "As long as this creature is on the battlefield, damage isn't removed from it during cleanup step."
+SVar:DBEffect:DB$ Effect | ConditionDefined$ Targeted | ConditionPresent$ Creature | RememberObjects$ Targeted | StaticAbilities$ PerpetualEffect | Name$ Melt Through's Perpetual Effect | Duration$ Permanent
+SVar:PerpetualEffect:Mode$ Continuous | Affected$ Card.IsRemembered | PresentZone$ Battlefield | AddHiddenKeyword$ Damage isn't removed from CARDNAME during cleanup steps. | Description$ As long as this creature is on the battlefield, damage isn't removed from it during cleanup step
```

Comment:
I don't think this PresentZone does what you want

Reply:
damage can only be dealt to player and Creatures or Planeswalkers on the Battlefield

Reply:
What you should do is get rid of useless params... (because Battlefield is default for AffectedZone anyway)